application/common/models/User.php

```diff
@@ -93,6 +122,22 @@ public function beforeDelete()
             }
         }
 
+        /*
+         * Delete email logs last in case other deletions trigger new emails
```

Comment:
So, _does_ deleting something else trigger an email? For example, if deleting an mfa record sends an email to the user, is that a problem when we're doing that in order to delete that user?

I think we're only going to delete users if they've been inactive for over a year, but I do wonder about the repercussions of emailing a user who is about to be purged from our system...

Reply:
Nice catch. So, to extrapolate this, would it be fair to say that no email should be sent to inactive users?

Reply:
Since this is a separate, preexisting issue, I'm going to proceed with merging this PR and create a separate issue/PR for the problem of sending an email to an about-to-be-deleted user.